src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -63,6 +64,34 @@ export class ThankYou extends Component {
         const trackingToken = getUrlParameter(params, 't') || undefined;
         const callerId = getUrlParameter(params, 'c') || undefined;
         this.removeTrackingGetArgs();
+        if (!calledState || !calledNumber) {
+            if (!calledState) {
+                //Send Sentry Breadcrumb
+                Sentry.addBreadcrumb({
+                    category: "Missing Thank You Arguments",
+                    message: "Thank you page accessed without state argument",
+                    level: Sentry.Severity.Info,
+                });
+            }
+            if (!calledNumber) {
```

Comment:
Like calledState calledNumber must be defined or an error will occur in findDistrictByStateNumber.  While we never had a state be provided but a number not provided we should handle this case.